### DISCUSSION TITLE
Way to remove certain sites

### DISCUSSION THREAD

**Discussion by PillowFood** [sherlock-project/sherlock#1354] [Ideas]
Wondering if someone's already suggested a way to remove certain sties during a serach.
I run into the issue of always getting Fiverr and Zhihu results even though those accounts don't exist.

If there's a functionality to eliminate those sites when searching, then my issue is resolved.
If not, maybe I could try my hand at coding it in and submitting a pull request. Thanks!

**Comment by Cherubrine** [comment]
I am trying to do the same. Have you found a solution yet?
Deleted the entries from data and sitelist but it still searches for them.

**Reply by rsb-23** [reply]
After deleting site data, use --local argument to run the command

**Comment by BlueCalex** [comment]
Says at the bottom of the main page on the project

_If some sites are failing due to connection problems (site is down, in maintenance, etc) you can exclude them from tests by creating a tests/.excluded_sites file with a list of sites to ignore (one site name per line)._

**Comment by TuriusUSA** [comment]
Is there a way to remove ALL of ones info from the sites??

**Reply by vehbiu** [reply]
You have to manually contact the website / delete your account. If you're putting personal information that you can't reveal on a website, that's on you.
